Features/Stories/Fieldtypes/CheckboxFieldType.feature

```diff
@@ -0,0 +1,78 @@
+        Given a Content Type with a "checkbox" Field exists
+        When I create a content of this Content Type
+        Then I should have a "checkbox" field
+
+    @javascript
+    Scenario: When editing a Content the label of a checkbox field must have the same name of the field type from the respective Content Type
+        Given a Content Type with a "checkbox" with "Name" "Do you agree?" exists
+        When I create a content of this Content Type
+        Then I should have a "Do you agree?:" label related with the "checkbox" field
+
+    @javascript
+    Scenario: The label of an mandatory checkbox field of a Content must have an * as suffix on the field's label's' name
+        Given a Content Type with a "required" "checkbox" with "Name" "Do you agree?" exists
+        When I create a content of this Content Type
+        Then I should have an "Do you agree?*:" label related with the "checkbox" field
+
+    ##
+    # Creating Content using a Content Type that has a Checkbox Field Type
+    ##
+    @javascript
+    Scenario: Publishing a valid checkbox Field works
+        Given a Content Type with a "checkbox" Field exists
+        When I create a content of this Content Type
+        And I check the Field Value
+        And I publish the content
+        Then the Content is successfully published
+
+    ##
+    # Update Content using a Content Type that has a Checkbox Field Type
+    ##
+    @javascript
+    Scenario: Updating a Checkbox field using a valid checkbox Field works
+        Given a Content Type with a "checkbox" Field exists
+        And a Content of this Content Type exists with "checkbox" Field Value checked
+        When I update this content unchecking the Field Value
+        And I publish the content
+        Then the Content is successfully published
+
+    ##
+    # Delete Content using a Content Type that has a Checkbox Field Type
+    ##
+    @javascript
+    Scenario: Deleting a content that has a Checkbox field
+        Given a Content Type with a "checkbox" Field exists
+        And a Content of this Content Type exists
+        When I delete this Content
+        Then the Content is successfully deleted
+
+    ##
+    # Viewing content that has a Checkbox fieldtype
+    ##
+    @javascript
+    Scenario: Viewing a Content that has a Checkbox fieldtype should show the expected value when the checkbox is checked
+        Given a Content Type with a "checkbox" Field exists
+        And a Content of this Content Type exists with "checkbox" Field Value checked
+        When I view this Content
+        Then I should see a field with value "Yes"
+
+    @javascript
+    Scenario: Viewing a Content that has a Checkbox fieldtype should show the expected value when the checkbox is unchecked
+        Given a Content Type with a "checkbox" Field exists
+        And a Content of this Content Type exists with "checkbox" Field Value unchecked

```

Comment:
The problem I see is when to do the publishing action to avoid creating multiple versions. 
To avoid that, we could either
- include/set all necessary properties in the same sentence (as we have now) or
- Have several sentences to create the object, but only to publish it in the end. 
With second approach, question is when do we know it's the end of the content definition? Until now, the action (the "when" part of BDD sentences)  is done with a "When I create|update|delete|view <something>". We could state that when we are going to do some action on the object then the object must exist and then we publish it..